### PR TITLE
Remove threads

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: Comms/CorrespondenceController.fs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+open Serilog
+open System.Net
+open System.Net.Http
+open System.Web.Http
+
+type CorrespondenceController() = 
+    inherit ApiController()
+    
+    member x.Get(profileIdOne : string, profileIdTwo : string) = 
+        Log.Information("Recieved get request for correspondence between profile 1: {profileIdOne} and profile 2 : {profileIdTwo}", profileIdOne, profileIdTwo)
```
Comment (csscottlogic) [inline Comms/CorrespondenceController.fs]: Received

Comment (jebbinSCL) [inline Comms/CorrespondenceController.fs]: Corrected

File: Comms/LastContactService.fs [hunk 2]
```diff
@@ -1,24 +1,42 @@
...
-    let lastCorrespondence = thread.Items |> Seq.maxBy (fun c -> c.Date)
-    {
-        ThreadId = thread.Id
-        Date = lastCorrespondence.Date
-        SenderId = lastCorrespondence.SenderId
-        ReceiverId = lastCorrespondence.ReceiverId
-    }
+let client = new AmazonDynamoDBClient(RegionEndpoint.EUWest1)
+let context = new DynamoDBContext(client)
+let pairIds (idOne : string) (idTwo : string) = String.concat "+" [ idOne; idTwo ]
```
Comment (csscottlogic) [inline Comms/LastContactService.fs]: Why not just ```idOne + idTwo```?

File: Comms/CorrespondenceController.fs [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+type CorrespondenceController() = 
+    inherit ApiController()
+    
+    member x.Get(profileIdOne : string, profileIdTwo : string) = 
+        Log.Information("Received get request for correspondence between profile 1: {profileIdOne} and profile 2 : {profileIdTwo}", profileIdOne, profileIdTwo)
+        let correspondence = getCorrespondence profileIdOne profileIdTwo
+        x.Request.CreateResponse(correspondence)
+    
+    member x.Post(correspondence) =
```
Comment (csscottlogic) [inline Comms/CorrespondenceController.fs]: Is this idempotent? Do we have the concept of a GUID for a correspondence item, i.e. "repo" (e.g. a Zoho account) + LUID for the "repo"?

File: Comms/LastContactService.fs (hunk 2 above)
Comment (jebbinSCL) [inline Comms/LastContactService.fs]: Just to make it clear where the split in the ids were if you ever need to look at the raw id.

**Pull request merged**